.github/workflows/build-and-upload-archives-upon-pushing-tags.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Upload Archives upon Creating Tags
+
+# Run this workflow every time a tag is created/pushed
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    # Name the Job
+    name: Build tagged commit and upload an archive
+    # Set the type of machine to run on
+    runs-on: ubuntu-latest
+    # timeout-minutes: 120
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # all history for all branches and tags
+
+      - name: Add upstream remote
+        if: github.repository != 'linkedin/venice'
```

Comment:
Do you think we should remove these two steps and add an
```if: github.repository = 'linkedin/venice'``` to the `build` job?

Reply:
Looks like we dont need them, let me remove them.